back-end/src/main/java/com/swnat/controller/CandidateController.java

```diff
@@ -46,5 +72,43 @@ public Candidate updateCandidate(@Valid @RequestBody Candidate candidate, @PathV
     public Candidate getCandidate(@PathVariable Long id) {
         return candidateService.getOne(id);
     }
-    
+
+    public String uploadImage(MultipartFile image) throws IOException {
+        String directory = System.getProperty("user.dir") + File.separator + "src" 
+        + File.separator + "main" + File.separator + "java" + File.separator + "com" + File.separator 
+        + "swnat" + File.separator + "profile_images";
+        Path mypath = Paths.get(directory);
+        if ( !Files.exists(mypath) ){
+            Files.createDirectory(mypath);
+            System.out.println("Directory created");
+        }
+        
+        StringBuilder builder = new StringBuilder();
+        builder.append(directory);
+        builder.append(File.separator);
+        builder.append(image.getOriginalFilename());
+
+        byte [] filebytes = image.getBytes();
+        Path path = Paths.get(builder.toString());
+        Files.write(path, filebytes);
+
+        return "http://localhost:8080/api/v1/uploads/" + image.getOriginalFilename();
```

Comment:
FYI. Yesterday we had a meeting with @veraivan about this method and how it just going to work for a dev environment only. He's working on a solution.

Reply:
Solution completed :+1:. That line of code was modified and now the image is built in the front-end with a new method added.

Reply:
> FYI. Yesterday we had a meeting with @veraivan about this method and how it just going to work for a dev environment only. He's working on a solution.

Solution completed :+1:. That line of code was modified and now the image is built in the front-end with a new method added.